media/pages/sk/podporte-nas/page.html

```diff
@@ -28,60 +28,61 @@ <h3>2% z daní</h3>
+        ročného zúčtovania zaplatených preddavkov na daň. Potom požiadajte zamestnávateľa
+        o vystavenie tlačiva <em>Potvrdenie o zaplatení dane</em>. Z tohto potvrdenia si viete
+        zistiť dátum zaplatenia dane a vypočítať 2% z Vašej zaplatenej dane
+        (to musí byť minimálne 3 €). Potrebné údaje vpíšte do tlačiva
+        <a href="{% page 'zamestnanci-chcemvediet.pdf' %}">Vyhlásenie o poukázaní 2%</a>. Najneskôr
+        do 30. apríla 2020 pošlite/doručte formuláre daňovému úradu v mieste vášho bydliska.
       </li>
     </ul>
   </div>
-</div>
-
-<h2>Aké sú naše výsledky?</h2>
-
-<div class="row">
-  <div class="col-sm-6">
+  <div class="col-md-6">
+    <h2>Aké sú naše výsledky?</h2>
     <p>
-      Od roku 2016 je portál Chcemvediet.sk prostriedkom k tomu, aby bežní ľudia mohli pohodlne podávať infožiadosti.
-      Medzi hlavné metriky, ktoré sledujeme, patrí počet <strong>registrovaných užívateľov</strong>, <strong>podaných
-      infožiadostí</strong> a počet <strong>inštitúcií v databáze</strong>.
+      Od roku 2016 je portál Chcemvediet.sk prostriedkom k tomu, aby bežní ľudia mohli pohodlne
+      podávať infožiadosti. Medzi hlavné metriky, ktoré sledujeme, patrí počet
+      <strong>registrovaných užívateľov</strong>, <strong>podaných infožiadostí</strong> a počet
+      <strong>inštitúcií v databáze</strong>.
     </p>
     <p>
       V roku 2017 sme sa spojili s ďalšími iniciatívnymi programátormi a tak vznikla platforma
-      <a href="https://verejne.digital">verejne.digital</a>. Na nej sme začali analyzovať prepojenia politikov a firiem
-      navzájom, a zobrazovať tieto prepojenia ako pekné grafy. Tiež sme analyzovali verejné obstarávania, kde hľadáme
-      podobné obstarávania, firmy, ktoré sa ich zúčastnili, a kontrolujeme, či sa cena obstarávania nevymyká doterajším
-      podobným obstarávaniam. Neskôr pribudol majetkový profil politikov na základe ich majetkových priznaní, spolu
-      s porovnaním voči údajom z katastra. Pred prezidentskými voľbami 209 sme doplnili majetkový profil kandidátov na
-      prezidenta.
+      <a href="https://verejne.digital">verejne.digital</a>. Na nej sme začali analyzovať prepojenia
+      politikov a firiem navzájom, a zobrazovať tieto prepojenia ako pekné grafy. Tiež sme
+      analyzovali verejné obstarávania, kde hľadáme podobné obstarávania, firmy, ktoré sa ich
+      zúčastnili, a kontrolujeme, či sa cena obstarávania nevymyká doterajším podobným
+      obstarávaniam. Neskôr pribudol majetkový profil politikov na základe ich majetkových priznaní,
+      spolu s porovnaním voči údajom z katastra. Pred prezidentskými voľbami 2019 sme doplnili
+      majetkový profil kandidátov na prezidenta.
     </p>
     Výročné správy:
     <ul>
-      <li><a href="{% page "vyrocna-sprava-2016.pdf" %}">Výročná správa za rok 2016</a></li>
+      <li><a href="{% page 'vyrocna-sprava-2016.pdf' %}">Výročná správa za rok 2016</a></li>
```

Comment:
Nechceli by sme sem namiesto beznych odrazok dat nejake ikonky. Pozri, ci v http://127.0.0.1:8000/styleguide/#icons nemame nejaku vhodnu, co by sa tu hodila.

Reply:
jj, vyzera to lepsie. Najviac sa mi paci ikonka pdf suboru, Tiez vhodne su spinka, maly sivy kruh, maly minus popripade nejaka ikonka download. S tym ze ikonky file-pdf, attach, download, by som spravil klikatelne, ikonky circle a minus neklikatelne.

```
<i class="chv-icon chv-icon-li chv-icon-lg icon-file-pdf chv-color-blue-ll"></i>

<i class="chv-icon chv-icon-li chv-icon-lg icon-attach chv-color-blue-ll"></i>
<i class="chv-icon chv-icon-li icon-circle"></i>
<i class="chv-icon chv-icon-li icon-minus chv-color-blue-ll"></i>
```

Reply:
PDF ukonka je super.